### DISCUSSION TITLE
[Android] Notification payload with click_action won't open the app

### DISCUSSION THREAD

**Discussion by cyberpolin** [invertase/react-native-firebase#5863] [Q&A]
when I send a notification with the `click_action` field e.g.

```
{
	"to": "user_fcm_token",
	"notification": {
		"title": "Check this Mobile (title)",
		"body": "Rich Notification testing (body)"
	},
	"priority": "high",
        "click_action": "inkind://history"
}
```

when the user tap the notification is the app is closed/foreground the notification is dismissed but the app won't open....

**Comment by mikehardy** [comment]
Hmm, not enough information to go on (e.g., https://stackoverflow.com/help/minimal-reproducible-example)
My guess is something to do with launchMode (maybe singleTop instead of singleTask, I'm not sure https://github.com/facebook/react-native/blob/aa4da248c12e3ba41ecc9f1c547b21c208d9a15f/template/android/app/src/main/AndroidManifest.xml#L17)
though it could be any of another number of libraries that are known to put launch activities in place of the react-native android MainActivity, and swallow intent extras as they pass the intent, e.g. https://github.com/crazycodeboy/react-native-splash-screen/issues/289#issuecomment-502406454
